lib/ledger_sync/adaptors/netsuite/department/ledger_serializer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module LedgerSync
+  module Adaptors
+    module NetSuite
+      module Department
+        class LedgerSerializer < NetSuite::LedgerSerializer
```

Comment:
Confusing, I know, but in this case you don't need the deserializer if they are the same.  It will infer the "opposite" of the serializer.  We still need to clean that up, but just FYI.